doc/2/controllers/server/get-config/snippets/get-config.test.yml

```diff
@@ -0,0 +1,7 @@
+name: server#getConfig
+description: Returns the current Kuzzle configuration.
+hooks:
+  before:
+  after:
+template: print-result
+expected: ^{.*}$
```

Comment:
Uh I am not convinced by the strength of this test :sweat_smile:

Reply:
The result is a very large object, I can change it to match some fields in deed

Reply:
Yes so at least it's would be different than an exception occurring :)